frontend/app/root.tsx

```diff
@@ -7,8 +7,9 @@ import { Links, LiveReload, Meta, Outlet, Scripts, ScrollRestoration, useLoaderD
 import { useTranslation } from 'react-i18next';
 import { getToast } from 'remix-toast';
 
+import { getSessionService } from './services/session-service.server';
 import { getFixedT, getLocale } from './utils/locale-utils.server';
```

Comment:
I know this isn't a part of your change, but can you modify this line, too?

```suggestion
import { getFixedT, getLocale } from '~/utils/locale-utils.server';
```

Reply:
modified